client/views/windows/settings.tpl

```diff
@@ -5,6 +5,23 @@
 	<h1 class="title">Settings</h1>
 
 	<div class="row">
+		<div class="col-sm-12">
+			<h2>
+				Settings synchronisation
+			</h2>
+			<p class="opt">
+				<input type="checkbox" name="syncSettings">
+				Synchronize settings with other clients. <br>
+				<strong>Warning</strong> Checking this box will override current client settings.
+			<p>
+			<p id="master-sync" class="opt">
+				No settings synced yet, sync settings from this client as base set?
```

Comment:
I'm not convinced on the value of this?

Under what circumstances would someone want to sync when there are no server settings and *not* set these as the ones currently synced.

Reply:
The first time someone does enable syncing there is nothing on the server yet. We could automatically sync the settings of that client but they might not have intended that. [See also this video](https://www.creesch.com/dump/vid/vid_5a3692e0339c4.mp4) for how it functions. 
This will basically give them the opportunity to go "Oh crap, I didn't want to sync this client but my other one". 

Basically done from the same reasoning as how I approached the advanced settings classification the other day :)

Reply:
I don't think you've convinced me of this one. I think it's a bit of a weird thing, but ... I'll let others say what they think. The rest I really like, I'm just not convinced about this 1 button press for when there are no synced settings and never again.

Reply:
As per the discussion on irc I'll remove the manual step and will instead display something like 

>  "No settings have been synced before. Enabling this will make the settings of this client the base settings."